pkg/features/kube_features.go

```diff
@@ -1235,6 +1235,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	genericfeatures.RemainingItemCount: {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
 
+	genericfeatures.ResilientWatchCacheInitialization: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
do we want to enable the feature straight away?

Reply:
yes - it was discussed in the KEP in more detail and directly with David, Joe and during SIG architecture meeting

Reply:
ok, thanks.